src/ci.rs

```diff
@@ -32,11 +32,18 @@ pub enum Platform {
     Macos,
     /// Emscripten
     Emscripten,
+    /// macOSM1
+    MacosM1,
```

Comment:
For future proof, how about naming this `MacosArm64`?

Reply:
Thanks. Renamed & fixed the failing CI